powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/SignECDSARequest.java

```diff
@@ -18,13 +18,23 @@
 
 package com.wultra.security.powerauth.client.model.request;
 
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import lombok.Data;
 
 /**
  * @author Petr Dvorak, petr@wultra.com
  */
 @Data
 public class SignECDSARequest {
+
+    @Schema(description = "Activation identifier")
+    @NotBlank(message = "Activation ID must not be empty when creating signature")
     private String activationId;
+
+    @Schema(description = "Signed data")
```

Comment:
The data are not signed yet, or are they?
```suggestion
    @Schema(description = "Data to be signed")
```